sample/github/legit.go

```diff
@@ -50,15 +49,23 @@ type GithubSecrets struct {
 	SecretToken string `json:"secretToken"`
 }
 
-// HandlePullRequest is invoked whenever a PullRequest is modified (created, updated, etc.)
-func (handler *GithubHandler) HandlePullRequest(payload interface{}, header webhooks.Header) {
-	glog.Info("Handling Pull Request")
+func (h *GithubHandler) handlePost(rw http.ResponseWriter, req *http.Request) {
+	body, err := ioutil.ReadAll(req.Body)
+	if err != nil {
+		panic(err)
+	}
+	rw.WriteHeader(200)
+	rw.Header().Add("content-type", "application/json")
```

Comment:
It looks like we never write anything to rw, so why set content-type?

Reply:
removed.